client/app/ip/ip/firewall/rule/add/ip-ip-firewall-rule-add.controller.js

```diff
@@ -12,7 +12,7 @@ angular.module("Module.ip.controllers").controller("IpFirewallAddRuleCtrl", ($sc
     };
 
     $scope.resetOptionField = function () {
-        if ($scope.rule.protocol !== "TCP") {
+        if ($scope.rule.protocol !== "tcp") {
```

Comment:
What about a service which would check if the value is empty and doesn't care if it's uppercase or lowercase ?

Reply:
it's going to be refacto @FredericEspiau, Fabien found a new bug ;)